README.md

```diff
@@ -4,6 +4,12 @@ Symfony Bundle Skeleton
 The "Symfony Bundle Skeleton" is an application for creating reusable Symfony bundles.
 Forked from [symfony/demo][1]
 
```

Comment:
This points to an other namespace repository. I think this file shouldn't be changed

Reply:
Yeah, copy and paste problem on my part.

Is there a generic way to say "get the github action badge from THIS repository"?  Otherwise, it seems like it's constantly pointing to a fork.

Reply:
Yes, I've added a new GitHub Action. You can modify it: https://github.com/msalsas/symfony-bundle-skeleton/blob/master/.github/workflows/symfony.yml

You can copy this code for the badge:

[![Symfony](https://github.com/msalsas/symfony-bundle-skeleton/actions/workflows/symfony.yml/badge.svg?branch=master)](https://github.com/msalsas/symfony-bundle-skeleton/actions/workflows/symfony.yml)

Reply:
Sorry, the code for the badge would be `[![Symfony](https://github.com/msalsas/symfony-bundle-skeleton/actions/workflows/symfony.yml/badge.svg)](https://github.com/msalsas/symfony-bundle-skeleton/actions/workflows/symfony.yml)`